indexer/editable_map_object.cpp

```diff
@@ -1,3 +1,5 @@
+#include "search/v2/postcodes_matcher.hpp"
```

Comment:
For those who don't believe: 
* add a unit test in indexer/indexer_tests/editable_map_object_test.cpp for EditableMapObject::ValidatePostcode()
* try to build it with static linkage
* receive
```
clang++ -ccc-gcc-name g++ -stdlib=libc++ -lprofiler -Wl,--gc-sections -Wl,-Bsymbolic-functions -Wl,-rpath,/home/y/Qt/5.5/gcc_64 -Wl,-rpath,/home/y/Qt/5.5/gcc_64/lib -o ../out/debug/MAPS.ME ../out/debug/tmp/qt/about.o ../out/debug/tmp/qt/create_feature_dialog.o ../out/debug/tmp/qt/draw_widget.o ../out/debug/tmp/qt/editor_dialog.o ../out/debug/tmp/qt/info_dialog.o ../out/debug/tmp/qt/main.o ../out/debug/tmp/qt/mainwindow.o ../out/debug/tmp/qt/osm_auth_dialog.o ../out/debug/tmp/qt/place_page_dialog.o ../out/debug/tmp/qt/preferences_dialog.o ../out/debug/tmp/qt/proxystyle.o ../out/debug/tmp/qt/qtoglcontext.o ../out/debug/tmp/qt/qtoglcontextfactory.o ../out/debug/tmp/qt/search_panel.o ../out/debug/tmp/qt/slider_ctrl.o ../out/debug/tmp/qt/update_dialog.o ../out/debug/tmp/qt/qrc_resources.o ../out/debug/tmp/qt/moc_about.o ../out/debug/tmp/qt/moc_create_feature_dialog.o ../out/debug/tmp/qt/moc_draw_widget.o ../out/debug/tmp/qt/moc_editor_dialog.o ../out/debug/tmp/qt/moc_info_dialog.o ../out/debug/tmp/qt/moc_mainwindow.o ../out/debug/tmp/qt/moc_osm_auth_dialog.o ../out/debug/tmp/qt/moc_place_page_dialog.o ../out/debug/tmp/qt/moc_preferences_dialog.o ../out/debug/tmp/qt/moc_search_panel.o ../out/debug/tmp/qt/moc_update_dialog.o   -L../out/debug -lmap -ldrape_frontend -lrouting -lsearch -lstorage -lindexer -ldrape -lplatform -leditor -lgeometry -lcoding -lbase -lfreetype -lexpat -lfribidi -ltomcrypt -ljansson -lprotobuf -losrm -lstats_client -lminizip -lsuccinct -lpugixml -loauthcpp -lopening_hours -lz -L/home/y/Qt/5.5/gcc_64/lib -lQt5OpenGL -L/usr/lib64 -lQt5Widgets -lQt5Gui -lQt5Network -lQt5Core -lGL -lpthread 
../../out/debug/libindexer.a(editable_map_object.o): In function `osm::EditableMapObject::ValidatePostCode(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)':
/mnt/linux-secondary-hdd/y/coding/omim/indexer/editable_map_object.cpp:252: undefined reference to `search::v2::LooksLikePostcode(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, bool)'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Makefile:222: recipe for target '../../out/debug/indexer_tests' failed
```
* ???
* PROFIT